main.go

```diff
@@ -121,7 +121,9 @@ func Router(server *socketio.Server) *mux.Router {
 	r.HandleFunc("/organizations/{id}/token-transactions", au.IsAuthenticated(orgs.GetTokenTransaction)).Methods("GET")
 	r.HandleFunc("/organizations/{id}/upgrade-to-pro", au.IsAuthenticated(orgs.UpgradeToPro)).Methods("POST")
 	r.HandleFunc("/organizations/{id}/charge-tokens", au.IsAuthenticated(orgs.ChargeTokens)).Methods("POST")
-	r.HandleFunc("/organizations/{id}/checkout-session", orgs.CreateCheckoutSession).Methods("POST")
+	r.HandleFunc("/organizations/{id}/checkout-session", au.IsAuthenticated(orgs.CreateCheckoutSession)).Methods("POST")
+	r.HandleFunc("/organizations/members/{mem_id}/addcard", au.IsAuthenticated(orgs.AddCard)).Methods("POST")
+	r.HandleFunc("/organizations/members/card/{id}", au.IsAuthenticated(orgs.DeleteCard)).Methods("DELETE")
```

Comment:
This /organizations/members/card/{id} is wrong. 
You are saying, in the context of organizations, in the context of members then in the context of this card resource. Then all you do is use just the card resource. So any one in any org that has access to a card id can initiate this request as long as they are logged in. We need to validated who is doing this action, do they have the right? do they belong to this org? are they active member?